### PR TITLE
Explore auto-generating the Swift interface for Danger

### PR DESCRIPTION
**Pull request by orta**
Uses the techniques in https://github.com/jpsim/SourceKitten/issues/405 to generate the Swift module interface directly, this matches https://danger.systems/js/reference.html / https://danger.systems/reference.html better.

### CODE REVIEW

File: Scripts/data/request.template.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# This file
```
Comment (f-meloni) [inline Scripts/data/request.template.yml]: This should be removed i think

Comment (orta) [reply Scripts/data/request.template.yml]: ah yeah, I was meant to finish that

**Comment by f-meloni** [comment]
should this maybe be added to rocket?

**Comment by orta** [comment]
Yeah, probably, I'm not sure of this is the end result or a partial one for now, so I'm happy to let it be a bit out of date right now as I figure out docs.

**Pull request merged**